doc/tutorial.md

```diff
@@ -195,24 +195,25 @@ minlenratio: 0.3
 - The pure attention mode requires to set the maximum and minimum hypothesis length (`--maxlenratio` and `--minlenratio`), appropriately. In general, if you have more insertion errors, you can decrease the `maxlenratio` value, while if you have more deletion errors you can increase the `minlenratio` value. Note that the optimum values depend on the ratio of the input frame and output label lengths, which is changed for each language and each BPE unit.
 - About the effectiveness of hybrid CTC/attention during training and recognition, see [2] and [3]. For example, hybrid CTC/attention is not sensitive to the above maximum and minimum hypothesis heuristics. 
 
-### Transducer
+## Transducer
```

Comment:
Transducer description should be the same header level as CTC/attention.
Please use `###`.
Also, the following headers should be `####`.